pubsub/pstest/fake.go

```diff
@@ -426,6 +426,14 @@ func (s *GServer) DeleteTopic(_ context.Context, req *pb.DeleteTopicRequest) (*e
 	if t == nil {
 		return nil, status.Errorf(codes.NotFound, "topic %q", req.Topic)
 	}
+	for _, sub := range s.subs {
+		if sub.deadletterTopic == nil {
```

Comment:
nit: prefer deadLetterTopic as dead lettering is two words. 

in general, s/deadletter/deadLetter/g
s/Deadletter/DeadLetter/g